docs/source/installation/versions.rst

```diff
@@ -2,22 +2,22 @@ Differences between Manim Versions
 ==================================
 
 While originally a single library, there are now three main versions of manim, 
-each with their own advantages, disadvantages and ideal use cases. 
+each with its own advantages, disadvantages, and ideal use cases. 
 It is important to understand these differences in order to select the best version 
 for your use case and avoid confusion arising from version mismatches.
 
 A brief history of Manim
 ************************
 
-Manim was originally created by Grant Sanderson as a personal project and 
-for use in his YouTube channel, `3Blue1Brown <https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw>`_. As his channel gained popularity, 
+Manim was originally created by Grant Sanderson as a personal project and for use in his YouTube channel, 
+`3Blue1Brown <https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw>`_. As his channel gained popularity, 
 many grew to like the style of his animations and wanted to use manim for their own projects. 
 However, as manim was only intended for personal use, 
 it was very difficult for other users to install and use it.
 
 In late 2019, Grant started working on faster OpenGL rendering in a new branch, 
-known as  the shaders branch. In mid 2020, a group of developers forked it into what is now the community edition; 
-this is the version which is documented by this website. 
+known as the shaders branch. In mid-2020, a group of developers forked it into what is now the community edition; 
+this is the version that is documented by this website. 
```

Comment:
`which` would've worked well here too...

Reply:
I think that would be more preferable

Reply:
How about: "this is the version documented on this website. "

Reply:
That works too!